src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-console.log('hello mate academy!');
+gallery();
+
+function gallery() {
+  const thumbs = document.body.querySelector('#thumbs');
+  const largeImage = document.body.querySelector('#largeImg');
+
+  thumbs.addEventListener('click', evt => {
```

Comment:
якщо в стрілочній функції відкриваємо фігурні дужки, то потрібні круглі дужки
```suggestion
  thumbs.addEventListener('click', (evt) => {
```
https://github.com/mate-academy/js_all-is-counted-DOM/compare/master...Kisilov-Vadim:Kisilov?expand=1